docs/deploy-development-rp.md

```diff
@@ -1,19 +1,31 @@
 # Deploy development RP
 
+## Why to use it?
+This is the **preferred** method. Is a fast way to have your own local development RP setup, while also having a functional cluster.
+It uses hacks scripts around a lot of the setup to make things easier to bootstrap and be more sensible for running off of your local laptop. 
+
+- Check the specific use-case examples where [deploying full RP service](https://github.com/Azure/ARO-RP/blob/master/docs/deploy-full-rp-service-in-dev.md) can be a better match.
+
 ## Prerequisites
 
 1. Your development environment is prepared according to the steps outlined in [Prepare Your Dev Environment](./prepare-your-dev-environment.md)
 
 ## Installing the extension
 
-1. Build the development `az aro` extension:
+1. Check the `env.example` file and copy it by creating your own:
+
+   ```bash
+   cp env.example env
+   ```
+
+2. Build the development `az aro` extension:
```

Comment:
With Markdown you can just leave them as 1's and if the indentation is correct they get rendered correctly. I think this is our general preference because we don't have to update numbers in future to re-order/add new points.

Ref: https://carpentries-incubator.github.io/markdown-intro/06-lists/index.html